server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -160,14 +151,6 @@ private void loadProps() {
 
       conf.setWalFolder(properties.getProperty("wal_dir", conf.getWalFolder()));
 
-      conf.setFlushWalThreshold(Integer
-          .parseInt(properties.getProperty("flush_wal_threshold",
-              Integer.toString(conf.getFlushWalThreshold()))));
-
-      conf.setForceWalPeriodInMs(Long
-          .parseLong(properties.getProperty("force_wal_period_in_ms",
-              Long.toString(conf.getForceWalPeriodInMs()))));
-
```

Comment:
Why no longer need to set these configs?

Reply:
I extract them into a separate method called `loadWALProps();` which will reduce duplicate codes because these parameters could be hot-loaded.